Jenkinsfile

```diff
@@ -95,6 +95,63 @@ def rpm_test_daos_test = '''me=\\\$(whoami)
                             trap 'set -x; kill -INT \\\$AGENT_PID \\\$COPROC_PID' EXIT
                             OFI_INTERFACE=eth0 daos_test -m'''
 
+def rpm_scan_pre = '''set -ex
+                      lmd_tarball='maldetect-current.tar.gz'
+                      if test -e "${lmd_tarball}"; then
+                        zflag="-z ${lmd_tarball}"
+                      else
+                        zflag=
+                      fi
+                      curl http://rfxn.com/downloads/${lmd_tarball} \
+                        ${zflag} --silent --show-error --fail -o ${lmd_tarball}
+                      nodelist=(${NODELIST//,/ })
+                      scp -i ci_key ${lmd_tarball} \
+                        jenkins@${nodelist[0]}:/var/tmp
+                      ssh -i ci_key jenkins@${nodelist[0]} "set -ex\n'''
+
+def rpm_scan_test = '''lmd_src=\\\"maldet-current\\\"
+                       lmd_tarball=\\\"maldetect-current.tar.gz\\\"
+                       rm -rf /var/tmp/\\\${lmd_src}
+                       mkdir -p /var/tmp/\\\${lmd_src}
+                       tar -C /var/tmp/\\\${lmd_src} --strip-components=1 \
+                         -xf /var/tmp/\\\${lmd_tarball}
+                       pushd /var/tmp/\\\${lmd_src}
+                         sudo ./install.sh
+                         sudo ln -s /usr/local/maldetect/ /bin/maldet
+                       popd
+                       sudo freshclam
+                       rm -f /var/tmp/clamscan.out
+                       sudo clamscan -d /usr/local/maldetect/sigs/rfxn.ndb \
+                                -d /usr/local/maldetect/sigs/rfxn.hdb -r \
+                                --exclude-dir=/usr/local/maldetect \
+                                --exclude-dir=/usr/share/clamav \
+                                --infected /etc /usr | \
```

Comment:
What if we've introduced a virus into a file that doesn't live in `/etc` or `/usr`?  Maybe we should just be scanning `/`.  Ultimately it doesn't matter which file it's in, if a virus exists in any file after we've installed DAOS (and it's components), we should be alerted as either we introduced it or it's in a package we installed on the base system and should be chased down.

Reply:
Fine tuning the Virus scan is something that the developers can do after we get this stage installed.

Starting the scan at / requires quite a lot of exclusions because all the special directories in it or it generates a lot of noise.

ClamAV is also not smart enough to auto-exclude its own virus signature databases from the scan, and will report them as infected files.

Reply:
Scan is now covering '/', and excluding /sys and the ClamAV / Maldet files.

Reply:
[Realizing that this particular issue is resolved in the most recent commit, but just to close out the conversation...]

I don't think it's realistic to expect that developers are going to know/remember to update the virus scan as a result of deciding to install something into a dir other than `/etc` or `/usr`.

Keep in mind, we are also responsible for all of the install paths of all of the components we package into RPMs.  Who is going to pay attention to all of those, every time there is an update to them to ensure they didn't do the same thing.